### DISCUSSION TITLE
Ability to refresh elements of a panel programmatically from a panel extension

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#21362] [Feature Requests]
### Summary

My panel extension deals with the DB.
On the same Dashboard, I have metrics.
How synchronize the metrics with the updates made on the DB by the panel extension?

### Basic Example

having some /refresh to post at the right time from the panel extension

### Motivation

Show synchronized results

### Detailed Design

something like api.refresh or api.post('/refresh ??

The same as the periodic refresh done every 10 seconds or so.
But accessible to the panel extension

### Must Have:
- A
- B

### Should Have:
- C

### Could Have:
- D

### Won't Have:
- E

### Drawbacks

None

### Alternatives

Periodic update. But it can't be every seconds.

### Adoption Strategy

None

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
